src/test/java/com/stripe/StripeTest.java

```diff
@@ -536,6 +535,80 @@ public void testDisputedCharge() throws StripeException, InterruptedException {
 	}
 
 	@Test
+	public void testRetrieveDispute() throws StripeException, InterruptedException {
+		int chargeValueCents = 100;
+		Charge disputedCharge = createDisputedCharge(chargeValueCents);
+		Dispute dispute = disputedCharge.getDispute();
+		Dispute retrievedDispute = Dispute.retrieve(dispute.getId());
+		assertEquals(dispute.getId(), retrievedDispute.getId());
+	}
+
+	@Test
+	public void testDisputeList() throws StripeException, InterruptedException {
+		createDisputedCharge(100).getDispute();
+		createDisputedCharge(150).getDispute();
+		createDisputedCharge(300).getDispute();
```

Comment:
I'm not sure that these three charge creations are necessary - there seem to be existing disputes on this account already. For now I have these here just to be extra safe, but let me know if it should be peaceful to remove them.